content/en/docs/topics/kubernetes_apis.md

```diff
@@ -123,15 +123,20 @@ Alternatively, you can follow these manual steps to perform an update of the API
 versions of a release manifest:
 
 - Get the name of the latest deployed release:
```

Comment:
Maybe we should prefix the above sentence with: "Depending on your configuration..."

And I guess we are not getting the name of the latest deployed release, but the "name of the secret or configmap associated with the latest deployed release"

Reply:
Good suggestion. Updated.